tests/postgres_tests/test_json.py

```diff
@@ -309,3 +309,18 @@ class CustomJSONField(forms.JSONField):
 
         field = CustomJSONField()
         self.assertIsInstance(field.widget, widgets.Input)
+
+    def test_already_converted_value(self):
+        field = forms.JSONField()
+        vals = [
+            ['a', 'b', 'c'],
+            {'a': 1, 'b': 2},
+            1,
+            True,
+            None,
+        ]
+        for val in vals:
+            try:
+                self.assertEqual(field.clean(val), val)
+            except ValueError:
```

Comment:
Shouldn't it be a `ValidationError` in this case? What about letting the original exception propagate instead?